src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -59,7 +52,7 @@ func NewClient(addr, appName string) *MobileClient {
 
 	hqfd := client.Configure(clientConfig.Client)
 
-	hqfdc := hqfd.DirectHttpClient()
+	hqfdc := hqfd.NewDirectDomainFronter()
```

Comment:
I think we actually want this to be a util.HttpClient. This using DDF appears to just be a bug in both lantern-android and lantern.